packaging/centos/datadog-agent.init

```diff
@@ -42,6 +42,7 @@ DD_OPT_PROC_REGEX="dogstatsd|jmxfetch|go-metro"
 # Source function library.
 . /etc/rc.d/init.d/functions
 
+[ -r /etc/default/datadog-agent ] && . /etc/default/datadog-agent
```

Comment:
on centos I think the convention is to source `/etc/sysconfig/datadog-agent` instead of `/etc/default/datadog-agent`